### PR TITLE
Fixes categories not visible when no answer from backend or list is null

### PR DESCRIPTION
**Pull request by ottpeter**

### CODE REVIEW

File: src/pages/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ const HomePage: React.FC<HomePageProps> = ({ isLoaded, withGamification }) => {
       .then((res) => res.json())
       .catch((err) => console.error("Error fetching user counts ", err));
 
-    if (roomsWithUserCount !== undefined) {
+    if (roomsWithUserCount.length > 0) {
```
Comment (bosi95) [inline src/pages/HomePage/HomePage.tsx:59]: what if roomsWithUserCount is undefined ?

Comment (ottpeter) [reply src/pages/HomePage/HomePage.tsx:59]: I think then the execution of this function will fail, and default value will be used.

Comment (bosi95) [reply src/pages/HomePage/HomePage.tsx:59]: would be nice to handle this case in the code, otherwise ok

**Review by bosi95 — APPROVED** [review]

File: src/pages/HomePage/HomePage.tsx (hunk 1 above)
Comment (ottpeter) [reply src/pages/HomePage/HomePage.tsx]: Added check for that.

**Pull request merged**